### ISSUE TITLE
suds equialent output data structure

### ISSUE THREAD

**Issue by tsillus** [mvantellingen/python-zeep#937]
src/zeep/wsdl/messages/soap.py:116-122 :

         if isinstance(result, xsd.CompoundValue):
             children = result._xsd_type.elements
             attributes = result._xsd_type.attributes
             if len(children) == 1 and len(attributes) == 0:
                 item_name, item_element = children[0]
                 retval = getattr(result, item_name)

Our company wants to switch from suds to zeep but we need to keep the suds like result data structure for backwards compatibility. For the moment removing this part of the code does the job for us. However I could take some time to put suds compatibility into zeep, if you'd like. So, do you have a preferred approach on how make this part of the code optional?

**Comment by pawel-czarnecki** [comment]
Exactly, I don't understand even why response structure is chaning when: len(children) == 1 and len(attributes) == 0 occurs ?
